samples/videoDecodeMem/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@
 #
 ################################################################################
 
-cmake_minimum_required (VERSION 3.5)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
why is this changed to 3.10?

Reply:
Version 3.5 will be deprecated and throw deprecation messages going forward. Version 3.10 is supported across all OS.